docs/framework/winforms/windows-forms-accessibility-improvements.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Windows Forms Accessibility Improvements"
+ms.date: "08/16/2019"
```

Comment:
- Can you add a `description: ` entry too, it should be around 100 characters long if possible. It should not repeat what the first paragraph in the article says.

- Also the date can be updated 😄

Reply:
Thank your for the review! Added description, fixed date.